### PR TITLE
Katherine Fitzpatrick -Testing

### PR DESCRIPTION
**Pull request by KatieFizzy**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,166 @@
...
+    const expected = 5;
+    const actual = helpers.subtractFive(10)
+
+    expect(actual).toEqual(expected);
+});
+
+test('should return true when two provided strings are same length', () => {
+    const expected = true;
+    const actual = helpers.areSameLength('test', 'bird')
+
```
Comment (kamry-bowman) [inline basic-javascript/project-1.spec.js]: for booleans, you want to check at least the false case and the true case

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,166 @@
...
+test('should return the provided number exponentially increased', () => {
+    const expected = 8;
+    const actual = helpers.raiseToPower(2,3)
+
+    expect(actual).toEqual(expected);
+});
+
+test('should return the provided number rounded to nearest integer', () => {
+    const expected = 10;
+    const actual = helpers.roundNumber(9.6)
```
Comment (kamry-bowman) [inline basic-javascript/project-1.spec.js:144]: could check round in both directions

File: basic-javascript/project-1.spec.js [hunk 3]
```diff
@@ -1,3 +1,166 @@
...
+});
+test('should return the provided string with hello added to the front', () => {
+    const expected = 'Hello webster!';
+    const actual = helpers.getGreeting('webster')
+
+    expect(actual).toEqual(expected);
+});
+test('should return the product of two numbers provided', () => {
+    const expected = 24;
+    const actual = helpers.getRectangleArea(8,3)
```
Comment (kamry-bowman) [inline basic-javascript/project-1.spec.js:184]: Could check how geometrical functions check negative values

File: basic-javascript/project-2.spec.js [hunk 4]
```diff
@@ -1,3 +1,128 @@
-const funcs = require('./project-2');
+const helpers = require('./project-2');
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:1]: Looks like there is still pending work for today on this one

**Review by kamry-bowman — COMMENTED** [review]
Nice work on this project!

File: basic-javascript/project-2.spec.js [hunk 5]
```diff
@@ -1,3 +1,134 @@
...
+    const expected = 'Guten Tag!';
+    const actual = helpers.greeting('German')
+//add edge case for languages not available
+// add edge case for when language isn't capitalized
+    expect(actual).toEqual(expected);
+});
+
+test('should return true when provided number equals five or ten', () => {
+    const expected = true;
+    const actual = helpers.isTenOrFive(10)
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:30]: Need to check for 5, and non-5 number

File: basic-javascript/project-2.spec.js [hunk 6]
```diff
@@ -1,3 +1,134 @@
...
+//add edge case for when number is not 5 or 10
+    expect(actual).toEqual(expected);
+});
+
+test('should return true when number is greater than twenty and is less than fifty', () => {
+    const expected = true;
+    const actual = helpers.isInRange(35) 
+
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:38]: Need to test less than 20, greater than 50

File: basic-javascript/project-2.spec.js [hunk 7]
```diff
@@ -1,3 +1,134 @@
...
+    expect(actual).toEqual(expected);
+});
+test('should return true when number is an integer', () => {
+    const expected = true;
+    const actual = helpers.isInteger(5)
+
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:44]: need to test for non-integers

File: basic-javascript/project-2.spec.js [hunk 8]
```diff
@@ -1,3 +1,134 @@
...
+    expect(actual).toEqual(expected);
+});
+
+test('should return fizzbuzz for number provided when divisible by three and five, buzz when divisible by five, and fizz when divisble by three', () => {
+    const expected = 'buzz';
+    const actual = helpers.fizzBuzz(20)
+
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:51]: need to test 5, 3, 15

File: basic-javascript/project-2.spec.js [hunk 9]
```diff
@@ -1,3 +1,134 @@
...
+    expect(actual).toEqual(expected);
+});
+test('should return true when number provided is prime', () => {
+    const expected = true;
```
Comment (kamry-bowman) [inline basic-javascript/project-2.spec.js:55]: need to test a bunch of primes and non-primes

**Review by kamry-bowman — COMMENTED** [review]
Nice work! Make sure you're taking the time to test enough to make sure functions are doing what they claim at a basic level
